src/table/table.jsx

```diff
@@ -27,6 +27,9 @@ const Table = React.createClass({
     selectable: React.PropTypes.bool,
     style: React.PropTypes.object,
     wrapperStyle: React.PropTypes.object,
+    headerTableInlineStyle: React.PropTypes.object,
```

Comment:
I kinda wanted the naming to be consistent with css classes, hence the `inline`.
The `Table` part is to avoid confusion with this:
```js
<Table bodyStyle={...}>
...
  <TableBody style={...}>
```
But there are more cases like this around the library. 

If you are ok with those I'll change the names and squash.